platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/iss/ConsensusHashManager.java

```diff
@@ -103,14 +113,12 @@ public class ConsensusHashManager {
     /**
      * Create an object that tracks reported hashes and detects ISS events.
      *
-     * @param platformContext        the platform context
-     * @param time                   provides the current wall clock time
-     * @param dispatchBuilder        responsible for building dispatchers
-     * @param addressBook            the address book for the network
-     * @param currentEpochHash       the current epoch hash
-     * @param currentSoftwareVersion the current software version
-     * @param ignoredRound           a round that should not be validated. Set to {@link #DO_NOT_IGNORE_ROUNDS} if all
-     *                               rounds should be validated.
+     * @param time                         provides the current wall clock time
+     * @param dispatchBuilder              responsible for building dispatchers
+     * @param addressBook                  the address book for the network
+     * @param currentEpochHash             the current epoch hash
+     * @param ignorePreconsensusSignatures If true, ignore signatures from the preconsensus event stream, otherwise
```

Comment:
you deleted the doc for `currentSoftwareVersion` and `ignoredRound`

Reply:
merge conflict gone sideways, restored the missing param docs